mirage/fixtures/external-storage-services.ts

```diff
@@ -0,0 +1,124 @@
+// TODO: needs iconUri and authUri
+export default [
+    {
+        id: 'bitbucket',
+        name: 'Bitbucket',
+        readOnly: true,
+        supportsCopy: false,
+        supportsUserSpecificRootFolder: false,
+        supportsFileVersioning: false,
+        supportsBulkDownload: false,
+        maxConcurrentDownloads: 2,
+        maxUploadMb: 0,
+    },
+    {
+        id: 'box',
```

Comment:
Actually, we might want to limit the number of these we have to just what we're expecting (i.e. let's just do box for now). Otherwise we're going to have duplicates in the UI when we have some from each source with conflicting names.